### PR TITLE
FAST: added Gitlab CI template

### PR DESCRIPTION
**Pull request by amgoogle**
- Added Gitlab CI Template
- Changed Identify Provider settings
- Fixed incorrect outputs
- Enabled Gitlab

### CODE REVIEW

File: fast/assets/templates/workflow-gitlab.yaml [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+  image:
+    name: registry.gitlab.com/gitlab-org/terraform-images/releases/1.1 
+
+variables:
+  FAST_OUTPUTS_BUCKET: ${outputs_bucket}
+  FAST_SERVICE_ACCOUNT: ${service_account}
+  FAST_WIF_PROVIDER: ${identity_provider}
+  SSH_AUTH_SOCK: /tmp/ssh_agent.sock
+  TF_PROVIDERS_FILE: ${tf_providers_file}
+  TF_VAR_FILES: ${tf_var_files == [] ? "''" : join("\n    ", tf_var_files)}
```
Comment (ludoo) [inline fast/assets/templates/workflow-gitlab.yaml:25]: nit: you could use [`indent`](https://www.terraform.io/language/functions/indent)

**Review by ludoo — APPROVED** [review]
This is great! stage 01 also supports repository definitions, we should enable Gitlab there too. It can be done here or in a subsequent PR whatever you prefer.

File: fast/assets/templates/workflow-gitlab.yaml (hunk 1 above)
Comment (amgoogle) [reply fast/assets/templates/workflow-gitlab.yaml:25]: 👍

Comment (amgoogle) [reply fast/assets/templates/workflow-gitlab.yaml:25]: Thanks for this. Leaving for the moment as we have the same here: https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/ab646f38ed6e0168c3fb9ea4a275dbb54e25b675/fast/assets/templates/workflow-github.yaml#L31

**Pull request merged**